kit/command_init.py

```diff
@@ -68,19 +76,39 @@ def append_to_rc(path: str, content: str) -> None:
             rc_file.write(line)
 
 
-def init_hekit(args):
-    """Initialize hekit"""
+def get_rc_file():
+    """ Return the correct file to add shell commands"""
     active_shell_path = Path(environment["SHELL"]).name
 
     if active_shell_path == "bash":
+        # if bash_profile file does not exist, try bashrc file
         rc_file = "~/.bash_profile"
+        if not Path(rc_file).expanduser().resolve().exists():
+            rc_file = "~/.bashrc"
     elif active_shell_path == "zsh":
         rc_file = ""
     else:
         raise ValueError(f"Unknown shell '{active_shell_path}'")
 
+    return rc_file
+
+
+def init_hekit(args):
+    """Initialize hekit"""
+    rc_file = get_rc_file()
     rc_backup_file = create_backup(rc_file)
     print("Backup file created at", rc_backup_file)
     remove_from_rc(rc_file)
-    append_to_rc(rc_file, content=f"PATH={args.hekit_root_dir}:$PATH")
+
+    # New lines that will be added in the rc_file:
+    # 1-Add hekit directory as part of environmental variable PATH
+    content = f"PATH={args.hekit_root_dir}:$PATH"
+
+    # 2-Register hekit link and hekit.py script to enable tab completion
+    if not util.find_spec("argcomplete") == None:
+        register_py = f'eval "$(register-python-argcomplete hekit.py)"'
+        register_link = f'eval "$(register-python-argcomplete hekit)"'
+        content = "\n".join([content, register_py, register_link])
+
```

Comment:
The test of argcomplete should not be in python it should be in POSIX shell if statement.
That way when a user decided afterwards that they suddenly do want tab completion they do not require to rerun hekit init.
```suggestion
    # 2-Register hekit link and hekit.py script to enable tab completion
    if not util.find_spec("argcomplete") == None:
        register_py = f'eval "$(register-python-argcomplete hekit.py)"'
        register_link = f'eval "$(register-python-argcomplete hekit)"'
        content = "\n".join([content, register_py, register_link])

```